app/forms/schools/cohorts/setup_wizard/success.rb

```diff
@@ -100,8 +100,7 @@ def what_changes_programme
         end
 
         def should_send_the_pilot_survey?
-          cohort.start_year == 2023 && FeatureFlag.active?(:cohortless_dashboard, for: school) &&
-            expect_any_ects? && current_user.induction_coordinator?
+          cohort.start_year == 2023 && expect_any_ects? && current_user.induction_coordinator?
```

Comment:
Can you check this is the right thing to do, looks like it will start ending the pilot survey to anyone setting up in this year, perhaps it needs to be removed instead?  Can you check with Kate and Nicola please.